resources/views/Staff/donations/subscriptions/index.blade.php

```diff
@@ -0,0 +1,228 @@
+                                    >
+                                        {{ __('common.edit') }}
+                                    </a>
+                                </td>
+                            </tr>
+                        @endforeach
+                    </tbody>
+                </table>
+                @if (! $subscriptions_active->count())
+                    <div class="margin-10">
+                        @lang('common.no-result')
+                    </div>
+                @endif
+                <br>
+                <div class="text-center">
+                    {{ $subscriptions_active->links() }}
+                </div>
+            </div>
+
+            <!-- Old VIP User-Records -->
+            <h3>Preivous VIP User Records</h3>
+            <div class="table-responsive">
+                <table class="table table-condensed table-striped table-bordered">
+                    <thead>
+                        <tr>
+                            <th>@lang('common.user')</th>
+                            <th>@lang('common.group')</th>
+                            <th>VIP Start</th>
+                            <th>VIP End</th>
+                            <th width="10%">@lang('common.action')</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach ($subscriptions_inactive as $subscription)
+                            <tr>
+                                <td>
+                                    <a href="{{ route('users.show', ['user' => $subscription->user->username]) }}">
+                                        {{ $subscription->user->username }}
+                                    </a>
+                                    @if ($subscription->user->is_donor && !in_array($subscription->user_id, $subscriptions_active_arr) == true)
+                                        <i class="fas fa-exclamation-circle text-orange" aria-hidden="true" data-toggle="tooltip" title="" data-original-title="User is still VIP but subscription has ended!"></i>
+                                    @endif
+                                </td>
+
+                                <td>
+                                    <span class="badge-user text-bold"
+                                        style="color:{{ $subscription->user->group->color }}; background-image:{{ $subscription->user->group->effect }};">
+                                        <i class="{{ $subscription->user->group->icon }}" data-toggle="tooltip"
+                                            data-original-title="{{ $subscription->user->group->name }}"></i>
+                                        {{ $subscription->user->group->name }}
+                                    </span>
+                                </td>
+
+                                <td>
+                                    {{ date('Y-m-d', strtotime($subscription->start_at)) }}
```

Comment:
```suggestion
                                    {{ $subscription->start_at->format('Y-m-d') }}
```

When we add the `datetime` cast to `start_at` in the model, it makes a carbon instance of that column. Change for the other one as well.

Reply:
ToDo

Reply:
Fixed in [c26f809](https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/3059/commits/c26f809a33651b0391750cdd8c14f9d21673b06a)